llama-index-packs/llama-index-packs-cohere-citation-chat/README.md

```diff
@@ -0,0 +1,49 @@
+# Cohere Citations Chat Engine Pack
+
+Creates and runs a custom `VectorStoreIndexWithCitationsChat` -- which provides the chat engine with documents/citation mode.
+See the documentation [here](https://docs.cohere.com/docs/retrieval-augmented-generation-rag) and [here](https://docs.cohere.com/docs/retrieval-augmented-generation-rag).
+
+## CLI Usage
```

Comment:
Can we mention that you can also directly install it if you don't want to look at/inspect the source code? `pip install llama-index-packs-cohere-citation-chat`

Reply:
Thank you